src/middlewared/middlewared/plugins/system.py

```diff
@@ -768,6 +771,7 @@ class SystemGeneralModel(sa.Model):
     stg_guihttpsredirect = sa.Column(sa.Boolean(), default=False)
     stg_language = sa.Column(sa.String(120), default="en")
     stg_kbdmap = sa.Column(sa.String(120))
+    stg_birthday = sa.Column(sa.DateTime(), nullable=True)
```

Comment:
You're also lacking database migration (`cd /usr/local/lib/python3.8/site-packages/middlewared && alembic revision --autogenerate`, it will appear in `alembic` subdir)

Reply:
I recall my `nullable` comment, seems we'll need `null` state until we are able to receive correct system time.

Reply:
Make sense, some.